doc/bql.rst

```diff
@@ -143,14 +143,23 @@ For those that are modeled, it specifies their statistical type.
 
 .. index:: ``CREATE POPULATION``
 
-``CREATE POPULATION [IF NOT EXISTS] FOR <name> WITH SCHEMA ( [GUESS STATTYPES
-FOR (<column(s)>)] [MODEL <column(s)> AS <stattype>] [IGNORE <column(s)>] )``
-
-   Create a population for *name* with a schema defined by guessing the
-   statistical types for some or all columns (which can be referred to by \*)
-   and/or explicitly modeling columns using a particular statistical type or
-   ignoring them. If more than one schema definition is used, they should be
-   separated by semicolons.
+``CREATE POPULATION [IF NOT EXISTS] FOR <name> WITH SCHEMA (*schema*)``
```

Comment:
Say `<schema>` here instead?  Within teletype text, `*schema*` does not italicize -- just leaves in asterisks.